solutions/observability/cloud/monitor-cloudtrail-logs.md

```diff
@@ -209,10 +209,11 @@ Navigate to {{kib}} and choose among the following monitoring options:
     :alt: Visualize CloudTrail logs with Disocver
     :::
 
-* **Visualize your logs with Logs explorer**
+* **Visualize your logs with Discover**
+
+    :::{image} ../../images/observability-firehose-cloudtrail-logsexplorer.png
+    :alt: Visualize CloudTrail logs with Discover
```

Comment:
You might be able to just delete this list item because the one above is also `Visualize your logs with Discover` so right now it just says it twice ([preview page](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/679/solutions/observability/cloud/monitor-cloudtrail-logs#firehose-cloudtrail-step-five)):

<img width="771" alt="Screenshot 2025-04-02 at 4 00 59 PM" src="https://github.com/user-attachments/assets/82a9b1f9-1168-48f3-bee4-9d05ac46744b" />

Reply:
Whoops, thanks for catching that!